test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -0,0 +1,73 @@
+import { assert } from 'chai'
+import { WalletFactory, XrplNetwork } from 'xpring-common-js'
+import { XrpError } from '../../src/XRP'
+import IssuedCurrencyClient from '../../src/XRP/issued-currency-client'
+
+// A timeout for these tests.
+// eslint-disable-next-line @typescript-eslint/no-magic-numbers -- 1 minute in milliseconds
+const timeoutMs = 60 * 1000
+
+// An address on TestNet that has a balance.
+const testAddressWithTrustLines =
+  'X7CSDUqZmWR7ggg9K2rTKDmEN53DH1x1j9MHK2foabFzapf'
+
+// An IssuedCurrencyClient that makes requests.
+const rippledGrpcUrl = 'test.xrp.xpring.io:50051'
+const rippledJsonUrl = 'http://test.xrp.xpring.io:51234'
+const issuedCurrencyClient = IssuedCurrencyClient.issuedCurrencyClientWithEndpoint(
+  rippledGrpcUrl,
+  rippledJsonUrl,
+  XrplNetwork.Test,
+)
+
+describe('IssuedCurrencyClient Integration Tests', function (): void {
+  // Retry integration tests on failure.
+  this.retries(3)
+
+  it('getTrustLines - valid request', async function (): Promise<void> {
+    this.timeout(timeoutMs)
+    // TODO: When SDK functionality is expanded, improve this integration test by first CREATING a trust line between two accounts,
+    // which will avoid the need for maintenance after a testnet reset.
+
+    // GIVEN a test address that has at least one trust line on testnet
+    // WHEN getTrustLines is called for that address
+    const trustLines = await issuedCurrencyClient.getTrustLines(
+      testAddressWithTrustLines,
+    )
+
+    // THEN there is a successful non-empty result
+    assert.exists(trustLines)
+    assert.isTrue(trustLines.length > 0)
```

Comment:
Why does this not also check the value of `trustLines`?

Reply:
That's a good question - basically, with integration tests, the data you're querying is always subject to change (for example, the testnet could be reset at any time)  in which case your tests are brittle.  However, once we have more methods in place around Issued Currencies, we can re-write this test to 
1) create a new, random faucet account(s)
2) establish a trustline or trustlines
3) query for them
all in the same widow of time, such that we can be sure of what we should be seeing.

In fact, I'll leave a TODO in there now